pss/pubsub.go

```diff
@@ -65,6 +68,8 @@ func (p *PubSub) Register(topic string, prox bool, handler func(msg []byte, p *p
 
 // Send sends a message using pss SendRaw
 func (p *PubSub) Send(to []byte, topic string, msg []byte) error {
+	metrics.GetOrRegisterCounter("pss.pubsub.send.num", nil).Inc(1)
+	defer metrics.GetOrRegisterResettingTimer("pss.pubsub.send", nil).UpdateSince(time.Now())
 	pt := message.NewTopic([]byte(topic))
-	return p.pss.SendRaw(PssAddress(to), pt, msg)
+	return p.pss.SendRaw(PssAddress(to), pt, msg, 20*time.Second)
```

Comment:
If we must include TTL for PSS, this should be configurable for any given `PubSub`, rather than hidden as it currently is. Please at least extract in a documented variable.

Reply:
Ok, I was not aware this Pubsub was used for other use cases.

Reply:
It isn't used currently for anything else, but this hardcoded value is rather hidden in its implementation. It'd be better if it was a `var` at the top of the package, or a config when you create a PubSub value.

Reply:
I have initialized PubSub in swarm.go with 20 seconds, maybe this value (and also outbox length should be configurable using flags, but I am not sure how to do it or if it makes sense do it now)